src/__tests__/a11yAPI.test.js

```diff
@@ -62,133 +62,255 @@ function Section() {
   );
 }
 
-test('getByA11yLabel, queryByA11yLabel, findByA11yLabel', async () => {
-  const { getByA11yLabel, queryByA11yLabel, findByA11yLabel } = render(
-    <Section />
-  );
+test('getByA11yLabel, getByLabelText, queryByA11yLabel, queryByLabelText, findByA11yLabel, findByLabelText', async () => {
+  const {
+    getByA11yLabel,
+    getByLabelText,
+    queryByA11yLabel,
+    queryByLabelText,
+    findByA11yLabel,
+    findByLabelText,
+  } = render(<Section />);
 
   expect(getByA11yLabel(BUTTON_LABEL).props.accessibilityLabel).toEqual(
     BUTTON_LABEL
   );
+  expect(getByLabelText(BUTTON_LABEL).props.accessibilityLabel).toEqual(
+    BUTTON_LABEL
+  );
   const button = queryByA11yLabel(/button/g);
   expect(button && button.props.accessibilityLabel).toEqual(BUTTON_LABEL);
+  const buttonAlias = queryByA11yLabel(/button/g);
```

Comment:
```suggestion
  const buttonAlias = queryByLabelText(/button/g);
```

Reply:
Thanks for catching these. I rushed this last commit and forgot to actually change the queries to their aliases 🥴

I'll have a better edit up soon.